device/workstation/w_darwin.go

```diff
@@ -38,9 +38,10 @@ func newWorkstation() WorkStation {
 }
 
 const diskSelectionTries = 3
+const writeAttempts = 3
 
 // Notifies user to chose a mount, after that it tries to write the data with `diskSelectionTries` number of retries
-func (d *darwin) WriteToDisk(img string) (progress chan bool, err error) {
+func (d *darwin) WriteToDisk(img string) (job *help.BackgroundJob, err error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 21 of function (*darwin).WriteToDisk() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->